desktop/menus/utils.js

```diff
@@ -14,10 +14,10 @@ const buildRadioGroup = ({ action, propName, settings }) => {
   };
 };
 
-const appCommandSender = (arg) => {
+const appCommandSender = (commandName = 'appCommand', arg) => {
```

Comment:
can we avoid adding optional arguments before required arguments? this can be pretty confusing especially since JS doesn't require a fixed arity when calling functions

Reply:
I think we can make it mandatory.